.golangci.yml

```diff
@@ -1,3 +1,6 @@
+run:
+  tests: false
```

Comment:
does this mean we're disabling it for tests?

Reply:
yes disabling linting for tests. Do we want linting for tests?

Reply:
Tests are important. Maybe even more so than the actual code